### PR TITLE
added the ability to check for unread messages with the 'i' key

### PR DESCRIPTION
**Pull request by mekhami**
Addresses Issue #109

Also two small PEP8 changes.

### CODE REVIEW

File: rtv/page.py [hunk 1]
```diff
@@ -412,8 +412,23 @@ def edit(self):
             s.catch = False
             self.refresh_content()
 
+    @BaseController.register('i')
+    def get_inbox(self):
+        """
+        Checks the inbox for unread messages and displays a notification.
+        """
+        try:
+            if len(list(self.reddit.get_unread(limit=1))) > 0:
```
Comment (michael-lazar) [inline rtv/page.py]: I think you need to move the praw call outside of the if-else statement, otherwise it may get called twice.

**Comment by mekhami** [comment]
@michael-lazar addressed that, which seems so obvious in hindsight. It also works twice as fast now. who knew.

**Comment by Brobin** [comment]
Is the plan to support actually viewing the inbox in the future? From what I see, this only checks if there is at least one new messages and shows it to you.

**Comment by mekhami** [comment]
Yeah, I hope to add more functionality like that, but for now, i just want
to keep rolling out mini-features and iterating over it. I think it's a
good strategy here.

**Comment by Brobin** [comment]
Cool. I was just clarifying because you said it addresses issue #109, but only does one part. Great work, I'm looking forward to this!

**Pull request merged**